Fix servicenow image upload (#23852)
demisto/content: Packs/ServiceNow/ReleaseNotes/2_5_6.md

#### Integrations
##### ServiceNow v2

> - Fixed an issue in **servicenow-upload-file** where images were uploded like files and missed preview.

Reviewer: ```suggestion
- Fixed an issue in **servicenow-upload-file** where images were missing preview when uploaded.
```
